packages/hpe_aruba_cx/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -1243,19 +1271,32 @@ processors:
       pattern: "Multicast L3 Bridge Control Forwarding entry with uuid %{aruba.instance.id} has no reference to a VLAN"
 
     # Loop Protect events (280x)
-    # https://www.arubanetworks.com/techdocs/AOS-CX/10.07/HTML/5200-8214/Content/events/LOOP-PROTECT.htm
+    # https://arubanetworking.hpe.com/techdocs/AOS-CX/10.15/HTML/elmrg/Content/events/LOOP-PROTECT.htm
   - grok:
       field: message
       tag: loop_protect_event_2801_through_2808
       description: "Logs Loop protect events"
-      if: "['2801','2802','2803','2804','2805','2806','2807','2808'].contains(ctx.event?.code)"
+      if: "['2801','2802','2803','2804','2805','2806','2807','2808','2809'].contains(ctx.event?.code)"
       patterns:
         - "^Ports TX %{DATA:aruba.loop.tx_port} and RX %{DATA:aruba.loop.rx_port} are disabled by Loop-protect after loop detection on VLAN %{GREEDYDATA:network.vlan.id}"
         - "^(Loop detected on port|Port) %{DATA:aruba.port}( is disabled by Loop-protection after loop detection)? on VLAN %{GREEDYDATA:network.vlan.id}"
         - "^Port %{DATA:aruba.port} enabled after disable time expired"
         - "^Port %{DATA:aruba.port} (added for|deleted from) loop-protection"
         - "^Loop-Protection stats cleared for port %{GREEDYDATA:aruba.port}"
         - "^Ports TX %{DATA:aruba.loop.tx_port} and RX %{DATA:aruba.loop.rx_port} are involved during TX port disabling"
+  - dissect:
+      field: message
+      tag: loop_protect_event_2810
+      description: ""
+      if: "ctx.event?.code == '2810'"
+      pattern: "Ports TX %{aruba.loop.tx_port} and RX %{aruba.loop.rx_port} are involved during RX port disabling"
+  - grok:
+      field: message
+      tag: loop_protect_event_2811
+      description: ""
+      if: "ctx.event?.code == '2811'"
+      patterns: 
+        - "Max vport limit %{DATA:aruba.limit.threshold} reached. Current vport %{NUMBER:aruba.limit.read_value:long}"
```

```suggestion
        - "^Max vport limit %{DATA:aruba.limit.threshold} reached. Current vport %{NUMBER:aruba.limit.read_value:long}"
```
Anchor to beginning?

---

yes, thanks. originally it was a dissect but I later changed it to Grok and missed this. Thanks for the catch. fixed